src/generic/json_minifier.h

```diff
@@ -11,7 +13,11 @@ class json_minifier {
   static error_code minify(const uint8_t *buf, size_t len, uint8_t *dst, size_t &dst_len) noexcept;
 
 private:
-  really_inline json_minifier(uint8_t *_dst) : dst{_dst} {}
+  ///Os3
+  really_inline json_minifier(uint8_t *_dst)
```

Comment:
Please remove these changes to the constructor. It will handled in a distinct PR. Remove the comment ///Os3

Reply:
done